app/javascript/components/rooms/room/Room.jsx

```diff
@@ -31,13 +36,25 @@ export default function Room() {
         </Col>
       </Row>
       <Row className="my-5">
-        <Col>
-          <h3>{room.name}</h3>
+        <Col className="col-xxl-8">
+          <Stack direction="horizontal" gap={2}>
+            <h1>{room.name}</h1>
+            { isRunning
+              && (
+                <Badge className="rounded-pill online-pill ms-2 text-success">
```

Comment:
Can we extract this into a shared component?
That will help us re-using it,

Reply:
It is changed into a component in the following PR #3938